sites/footers/deprecated/footers-v2-en.html

```diff
@@ -0,0 +1,117 @@
+---
+{
+"layout": false,
+"title": "Content page including complete footer version 2.0 (deprecated)",
+"language": "en",
+"breadcrumbs": [
+	{ "title": "Footer", "link": "sites/footers/footers-en.html" }
+],
+"altLangPage": "footers-v2-fr.html",
+"secondlevel": false,
+"dateModified": "2022-06-17",
+"share": "true"
```

Comment:
Page details (Share this page and date modified) don't show on this page (EN & FR).